tests/test_healpix_padding.py

```diff
@@ -100,3 +100,26 @@ def test_healpix_pad(backend, device):
     out.mean().backward()
     assert out.shape == (n, ntile, nside + padding * 2, nside + padding * 2)
     assert x.grad.shape == (n, ntile, nside, nside)
+
+
+@pytest.mark.parametrize("nchannels", [128, 129])
+@pytest.mark.parametrize("dtype", [torch.float32, torch.float64, torch.bfloat16, torch.float16])
+def test_healpix_pad_cuda_channels_last(nchannels, dtype):
+    if not torch.cuda.is_available():
+        pytest.skip("CUDA not available.")
+
+    ntile = 12
+    nside = 32
+    padding = 1
+    n = 3
+    x = torch.randn([n, ntile, nchannels, nside, nside], device="cuda", dtype=dtype)
```

Comment:
Maybe worth adding another alignment test, something like:
```python
x = torch.randn([n * ntile * nchannels * nside * nside + 1], device="cuda", dtype=dtype)
x = x[1:].reshape([n, ntile, nchannels, nside, nside])
```
The pointer for this `x` ends up being unaligned by 2 bytes if `dtype` is `torch.bfloat16`:
 ```python
>>> x.__cuda_array_interface__['data'][0] % 8
2
```